addons-l10n/en/editor-delete-button.json

```diff
@@ -0,0 +1,7 @@
+{
+  "editor-delete-button/button-text": "Delete",
```

Comment:
This should say "Delete Project".

Reply:
I don't think "project" needs to be capitalized.